app/src/components/code/PythonBlock.tsx

```diff
@@ -24,7 +24,7 @@ export function PythonBlock(props: PythonBlockProps) {
       {...props}
       basicSetup={{
         lineNumbers: false,
-        foldGutter: true,
+        foldGutter: false,
```

Comment:
does this just allow for collapsing blocks?

Reply:
It gets weird of a weird border visible only in light mode, I don't think there was any functionality change but I could be wrong